dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/domain/Event.java

```diff
@@ -96,10 +86,6 @@ public class Event implements TrackerDto, Serializable {
 
   @JsonProperty private User assignedUser;
```

Comment:
Can users set `completedBy/At`?

Reply:
I asked this myself and then forgot to go forward with that.
The `completedBy` in my head shouldn't be set by the user, you shouldn't be able to do stuff on behalf of others.
The `createdAt` could be something that the user needs to set. I asked @vgarciabnz how it is expected to work when Android is synching.

Reply:
I had a chat with Victor another with Ameen and Joakim. `completedAt` could be set by the user so I will put it back for `Enrollment` and leave it as it is for `Event`.
I am going to do it in next PR though